test/e2e/e2e_test.go

```diff
@@ -151,6 +151,55 @@ func TestDisasterRecoveryAll(t *testing.T) {
 	testDisasterRecovery(t, 3)
 }
 
+// TestPauseControl tests the user can pause the operator from controlling
+// an etcd cluster.
+func TestPauseControl(t *testing.T) {
+	f := framework.Global
+	testEtcd, err := createEtcdCluster(f, makeEtcdCluster("test-etcd-", 3))
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer func() {
+		if err := deleteEtcdCluster(f, testEtcd.Name); err != nil {
+			t.Fatal(err)
+		}
+	}()
+
+	names, err := waitUntilSizeReached(f, testEtcd.Name, 3, 60)
+	if err != nil {
+		t.Fatalf("failed to create 3 members etcd cluster: %v", err)
+	}
+
+	testEtcd.Spec.Paused = true
+	if testEtcd, err = updateEtcdCluster(f, testEtcd); err != nil {
+		t.Fatalf("failed to pause control: %v", err)
+	}
+
+	// TODO: this is used to wait for the TPR to be updated.
+	// TODO: make this wait for reliable
+	time.Sleep(5 * time.Second)
```

Comment:
The full TPR update cycle has two phase. API server updates TPR, API server sends TPR updates to the watcher in our operator, the operator sends the update to the cluster monitor routine, the monitor routine finally update its TPR cache (the spec). We need to wait for the full cycle. Basically we can remove this once our operator writes status back. Now we do not.